internal/netconf/testdata/machine.yaml

```diff
@@ -1,69 +1,46 @@
 ---
 hostname: machine
 networks:
-  # === Tenant Network (private=true)
-    # [IGNORED]
   - asn: 4200003073
-    # [IGNORED in case of private network]
     destinationprefixes: []
-    # For Machine: Used to set the loopback ips.
     ips:
       - 10.0.17.2
-    # [IGNORED in case of private network]
     nat: false
-    # [IGNORED in case of private network]
```

Comment:
Comments like this helps other to understand the WHY in the code. To me it is not a plus to remove those comments.